drivers/util.py

```diff
@@ -886,7 +886,9 @@ def handler(signum, frame):
     signal.signal(signal.SIGALRM, handler)
     signal.alarm(timeoutseconds)
     try:
-        function( * arguments)
+        result = function(*arguments)
+        signal.alarm(0)
```

Comment:
If this needs to be executed on both success and failure cases, it might be nicer for this to go into a `finally:` section

Reply:
Right. I hesitated. :) I will repush!